library/ssl_tls13_generic.c

```diff
@@ -2272,16 +2278,26 @@ static int ssl_read_certificate_verify_parse( mbedtls_ssl_context* ssl,
        return( MBEDTLS_ERR_SSL_BAD_HS_CERTIFICATE_VERIFY );
     }
 
-    MBEDTLS_SSL_DEBUG_BUF( 3, "verify hash", verify_hash, verify_hash_len );
-
-    ret = mbedtls_pk_verify( &ssl->session_negotiate->peer_cert->pk,
+    mbedtls_pk_rsassa_pss_options* opts_ptr = NULL;
+    mbedtls_pk_rsassa_pss_options opts;
```

Comment:
Minor: While we do support C99, most of the library places variable declarations at the beginning of the scope, and I'd prefer to stick that here, too.